lib/components/SDropdownSectionFilter.vue

```diff
@@ -95,6 +95,10 @@ function handleClick(option: DropdownSectionFilterOption, value: string | number
   border-bottom: 1px solid var(--c-divider-light);
   padding: 8px;
   background-color: var(--c-bg-elv-up);
+
+  .dark & {
```

Comment:
I don't think we need this. `--c-bg-elv-up` in dark mode is same as `--c-bg-mute` 👀

Reply:
Ah,  I'll fix it.